apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/disable-core-nux.js

```diff
@@ -24,4 +24,10 @@ subscribe( () => {
 			openedManually: true,
 		} );
 	}
+	if ( select( 'core/edit-site' )?.isFeatureActive( 'welcomeGuide' ) ) {
+		dispatch( 'core/edit-site' ).toggleFeature( 'welcomeGuide' );
+		dispatch( 'automattic/wpcom-welcome-guide' ).setShowWelcomeGuide( true, {
+			openedManually: true,
```

Comment:
In the case of initial load, I don't think it makes sense for `openedManually` to be `true`.

I think this is solved for the post editor by lines 8-10:
```
	if ( select( 'core/edit-post' )?.isFeatureActive( 'welcomeGuide' ) ) {
		dispatch( 'core/edit-post' ).toggleFeature( 'welcomeGuide' );
	}
	unsubscribe();
```
And we should probably add this for this `edit-site` version as well. It looks like that first subscribe block is meant to handle the initial load (then unsubscribe), while the second subscribe block is meant to remain and handle the later actions of the user trying to open the guide manually (hence the `openedManually: true` addition there).

So right now this works, but on load we are dismissing the core guide and setting the a8c guide to open (when it is already open) with the added `openedManually` prop. As opposed to just dismissing the core guide and leaving the a8c one alone.